File: test/e2e/regressions/python_project_donotopen/pipelines/PYTHON_PIPELINE_CONFIG/code/python_pipeline_config/graph/Reformat_1.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from prophecy.libs import typed_lit
from python_pipeline_config.config.ConfigStore import *
from python_pipeline_config.udfs.UDFs import *

def Reformat_1(spark: SparkSession, in0: DataFrame) -> DataFrame:
    return in0.select(
        col("customer_id"), 
        concat(col("first_name"), lit(Config.c_string), lit(Config.c_int), lit(Config.c_float)).alias("first_name"), 
        col("last_name"), 
        col("phone"), 
        col("email"), 
        col("country_code"), 
        col("account_open_date"), 
        col("account_flags")
    )

File: test/project_build_tool/scala/pipelines/Automated-scala-PBT-1669640932953/code/src/main/scala/io/prophecy/pipelines/automatedscalapbt1669640932953/graph/Reformat_1.scala
package io.prophecy.pipelines.automatedscalapbt1669640932953.graph

import io.prophecy.libs._
import io.prophecy.pipelines.automatedscalapbt1669640932953.config.ConfigStore._
import io.prophecy.pipelines.automatedscalapbt1669640932953.udfs.UDFs._
import io.prophecy.pipelines.automatedscalapbt1669640932953.udfs._
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object Reformat_1 {

  def apply(spark: SparkSession, in: DataFrame): DataFrame =
    in.select(
      concat(col("first_name"),
             col("last_name"),
             lit(Config.c_string),
             lit(Config.c_int),
             lit(Config.c_boolean)
      ).as("col1"),
      col("customer_id"),
      col("first_name"),
      col("last_name"),
      col("phone"),
      col("email"),
      col("country_code"),
      col("account_open_date"),
      col("account_flags")
    )

}
